docker/datahub-upgrade/datahub-upgrade.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd $DIR && docker pull acryldata/datahub-upgrade:head && docker run --env-file ./env/docker.env --network="datahub_network" acryldata/datahub-upgrade:latest "$@"
\ No newline at end of file
+cd $DIR && docker pull acryldata/datahub-upgrade:head && docker run --env-file ./env/docker.env --network="datahub_network" acryldata/datahub-upgrade:head "$@"
```

Comment:
could you use a common variable e.g. 
`DOCKER_IMAGE=acryldata/datahub-upgrade:head`
followed by
`cd $DIR && docker pull ${DOCKER_IMAGE} && docker run --env-file ./env/docker.env --network="datahub_network" ${DOCKER_IMAGE} "$@"`

Reply:
done!